src/main/java/com/iexec/sms/secret/MeasuredSecretService.java

```diff
@@ -110,9 +119,15 @@ private void countStoredSecrets() {
             final Long count = storedSecretsCountGetter.get();
             log.debug("Counting secrets [type:{}, count:{}]", secretsType, count);
             storedSecretsCount.set(count);
+            if (null != cacheSecretService) {
+                cachedSecretsCount.set(cacheSecretService.count());
+            } else {
+                cachedSecretsCount.set(0);
+            }
```

Comment:
```suggestion
            final cacheCount = cacheSecretService != null ? cacheSecretService.count() : 0;
            cachedSecretsCount.set(cacheCount);
```
or maybe better, replace CacheService by a LongSupplier
The function is CacheSecretService::count or a lambda returning 0L for enclave challenges

Reply:
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/function/LongSupplier.html